# Generated by Django 2.0 on 2018-04-22 09:06

import datetime
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('accounts', '0014_auto_20180422_1135'),
    ]

    operations = [
        migrations.AlterField(
            model_name='errorlogging',
            name='error_time',
            field=models.DateTimeField(default=datetime.datetime(2018, 4, 22, 12, 6, 40, 88462)),
        ),
        migrations.AlterField(
            model_name='userprofile',
            name='phonenumber',
            field=models.IntegerField(blank=True, null=True, validators=[django.core.validators.MaxValueValidator(99999999999999)]),
        ),
    ]
